mlkem/zetas.i

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2024 The mlkem-native project authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
```

Comment:
This should be an `.h` file, not `.i`? I think `.i` is typically used for snippets of assembly.